dashboard/app/models/levels/bubble_choice.rb

```diff
@@ -230,14 +231,20 @@ def setup_sublevels(sublevel_names)
     reload
   end
 
+  # Some BubbleChoice sublevels also have a contained level
+  def self.level_for_progress_for_sublevel(sublevel)
```

Comment:
I could use some input on whether this is the right place for this method. It could be in the level model, since sublevel is just a level

Reply:
I think you could either keep this method where it is, or get rid of it and call `get_level_for_progress` instead.

Reply:
Yeah, I did consider that. But there is logic about whether the level is a bubble choice in that method, and I thought that could mislead engineers to think that it would be possible to have a bubble choice as a sublevel. Do you think that's a concern?

Reply:
good point. I think the way you have it is better since it seems easier to understand. 👍

Reply:
if we ended up with arbitrarily deeply nested levels.. then we might want to consider a recursive solution. but we're trying hard to stay out of that situation.